schema_based_on_diagram.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE patients (
+ id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+ name VARCHAR(100),
+ date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  admitted_at timestamp NOT NULL,
+  patient_id INT, 
+  status VARCHAR NOT NULL,
+  FOREIGN KEY (patient_id) REFERENCES patients (id) ON DELETE CASCADE
```

Comment:
- [ ] all FOREIGN KEYS should have indexes created:

CREATE INDEX ON medical_histories (patient_id);